File: problems/algorithms/function-prime-nth/php/response/spec/prime-nth.spec.php
<?php

require __DIR__ . '/../src/prime-nth.php';

describe('Number Tools', function () {
  it('looking for the first 4 prime number', function () {
    expect(nthPrime(4))->toBe(7);
  });

  it('looking for the first 6 prime number', function () {
    expect(nthPrime(6))->toBe(13);
  });
});

File: src/pages/problems/function-prime-nth/_codes/ruby/code/prime-nth.spec.rb
require_relative 'prime-nth.rb'

describe 'Number Tools' do
  context 'looking for the first ' do
    it '4 prime number' do
      expect(nthPrime(4)).to eql(7);
    end
    it '6 prime number' do
      expect(nthPrime(6)).to eql(13);
    end
  end
end
